reconcile/utils/saasherder.py

```diff
@@ -334,6 +334,31 @@ def _additional_resource_process(resources, html_url):
                         'could not add html_url annotation to' +
                         resource['name'])
 
+    @staticmethod
+    def _parameter_required_and_absent(
+            parameter_name, consolidated_parameters, template):
+        """Is a parameter is required in the template but unspecified?
+
+        TODO: This is currently "parameter *named* and absent" -- i.e. we
+        don't care about `required: true`. This is for backward compatibility;
+        but should eventually be fixed.
```

Comment:
i'm not sure this statement is true. i think it's enough to name the parameter. most services just use the value `latest` and not make it required, which helps them deploy the template in their development process.

Reply:
I renamed the method and took out the TODOs.

(Nitpick: If they specify the value, `required` is redundant/unnecessary and wouldn't trigger this function anyway. But after poking around, I think I grok why it will *never* be possible to track down all the template files in existence and make sure they've specified `required: true` for `IMAGE_TAG`.)

Reply:
EXACTLY! if we can't automate it, we can't enforce it.